lib/builder.js

```diff
@@ -69,6 +69,8 @@ function combineMetadataAndValues(valuesObject, valuesMetadata) {
         // Set the value from actual object
         param.value = valuesObject[paramIndex].value;
         param.type = valuesObject[paramIndex].type;
+        // TODO(miguelaeh): Hack to avoid render parameters with dots in keys into the schema. Must be removed once fixed
+        param.schema = valuesObject[paramIndex].schema;
```

Comment:
I understand we are taking the value from the one defined in [parser.js](https://github.com/bitnami-labs/readme-generator-for-helm/pull/43/files#diff-2f45270c7389933f71b380538ff52e106ee5cb66a0b573d9967467dfdbc11c02) but I'm missing how the render is avoiding this parameter so it is not added to the schema.

Reply:
Because by default, all parameters are rendered into the schema. Setting the property to the value of `valuesObject[paramIndex].schema`, it will ship a boolean from the parse step in which there is a similar hack setting it to false until we refactor the schema-generation

Reply:
Thanks for the explanation, LGTM!